inspirehep/modules/migrator/ext.py

```diff
@@ -35,4 +36,5 @@ def __init__(self, app=None):
     def init_app(self, app):
         app.cli.add_command(migrate)
         app.cli.add_command(migrator)  # Deprecated
+        app.add_url_rule('/mirror', 'get_legacy_mirror_errors', view_func=get_legacy_mirror_errors)
```

Comment:
we don't need to add the rule here :) We are using ``entry_point``s, you can register your blueprint in the ``api_app`` (which has the ``/api`` prefix) like in this line https://github.com/inspirehep/inspire-next/blob/master/setup.py#L208